lib/UA.js

```diff
@@ -256,6 +259,39 @@ module.exports = class UA extends EventEmitter
 
     return message;
   }
+  
+  /**  
+   * Arbitrary client transaction 
+   */
+  sendRequest(method, target, params, headers, body, handlers, credential)
```

Comment:
Adding this method here is out of the scope of the PR. New classes should make use of `RequestSender` and `Dialog` internally to send requests.

Reply:
I can inline here sendRequest method body.


I don't know how to use Dialog class here (should be checked).
In the Notifier class I  implemented "dialog interface":
dialog.id and dialog.incomingRequest.